### PR TITLE
17 dumb web app content

### PR DESCRIPTION
**Pull request by leoNord462**

### CODE REVIEW

File: backbone-frontend/src/Content/Content.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            <Card onClick={() => handleCardClick(doc)}>
+              <p>Titre: {doc.title}</p>
+              <p>Score: {doc.bbscore}</p>
+              <p>Date de création: {doc.creation_date}</p>
+            </Card>
+          </Col>
+        ))}
+      </Row>
+      <Modal
+        visible={modalVisible}
```
Comment (leoNord462) [inline backbone-frontend/src/Content/Content.js:56]: visible is deprecated, use open instead

File: backbone-frontend/src/Content/Content.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      } else {
+        return b.bbscore - a.bbscore;
+      }
+    });
+    setDocs(sortedDocs);
+    setSortAscending(!sortAscending);
+  };
+
+  return (
+    <div>
```
Comment (leoNord462) [inline backbone-frontend/src/Content/Content.js:40]: You don't need to use a <div> for this. Just use <> your-content </>

File: backbone-frontend/src/Content/BBdocs.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
+export const BBdocs =[
```
Comment (leoNord462) [inline backbone-frontend/src/Content/BBdocs.js:1]: In order to respect the CamelCase standard for React, this should be renamed BBDocs

File: backbone-frontend/src/test.json
Comment (leoNord462) [inline backbone-frontend/src/test.json:1]: This file is useless

File: backbone-frontend/src/Content/Content.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+          </Col>
+        ))}
+      </Row>
+      <Modal
+        visible={modalVisible}
+        onCancel={handleModalClose}
+        footer={null}
+        width={800}
+      >
+        {selectedDoc && (
```
Comment (leoNord462) [inline backbone-frontend/src/Content/Content.js:61]: This is weird. Instead, you could create the modal based on a doc

**Review by leoNord462 — COMMENTED** [review]
Nice work, few things to change

File: backbone-frontend/src/Content/Content.js (hunk 2 above)
Comment (BAAAL00) [reply backbone-frontend/src/Content/Content.js:40]: fait

File: backbone-frontend/src/Content/BBdocs.js (hunk 3 above)
Comment (BAAAL00) [reply backbone-frontend/src/Content/BBdocs.js:1]: done

File: backbone-frontend/src/Content/Content.js [hunk 5]
```diff
@@ -48,6 +49,7 @@ const MyComponent = () => {
               <p>Titre: {doc.title}</p>
               <p>Score: {doc.bbscore}</p>
               <p>Date de création: {doc.creation_date}</p>
+              <OpenURLButton url={doc.file_path} name={"Ouvrir l'URL"}/>
```
Comment (leoNord462) [inline backbone-frontend/src/Content/Content.js:52]: Maybe you could easily use AntButton href property 
https://ant.design/components/button

File: backbone-frontend/src/Content/URLButton.js
Comment (leoNord462) [inline backbone-frontend/src/Content/URLButton.js:1]: See above comment

**Pull request merged**